fixtures/initialise_tables.sql

```diff
@@ -0,0 +1,74 @@
+-- Delete tables
+DROP TABLE IF EXISTS tags_challenges CASCADE;
+DROP TABLE IF EXISTS challenges CASCADE;
+DROP TABLE IF EXISTS tags_organisations CASCADE;
+DROP TABLE IF EXISTS organisations CASCADE;
+DROP TABLE IF EXISTS people CASCADE;
+
+-- TAGS --
```

Comment:
Deleting the tags and recreating them inside the challenges plugin will create some conflict if when we will use the challenges plugin with the tags plugin. If we need some tags to test the challenges plugin can we just register the tags plugin which will create the right tables?